### ISSUE TITLE
Server crash (IllegalStateException: Not a JSON Object)

### ISSUE THREAD

**Issue by jburrows82** [TheTemportalist/MorphAdditions#9]
java.lang.IllegalStateException: Not a JSON Object: "drop|6000,12000|minecraft:egg:0"

http://pastebin.com/xvrLg7rc

**Comment by HyperMan19680** [comment]
I can confirm this.

**Comment by TheTemportalist** [comment]
Yeah. I discovered that. Then fixed it locally. Then discovered an issue with morph. And am currently asking iChun why he does things.

**Comment by TheTemportalist** [comment]
https://github.com/iChun/Morph/issues/755

**Comment by TheTemportalist** [comment]
As far as I can tell, it looks like Morph Additions is out of the game until 1.8 (unless I can figure out some hackery)

**Comment by Tsunayoshi** [comment]
I have the same problem, as I updated my pack. Does the MorphAdditions-1.7.10-0.0.9 version with the Origin-1.7.10-3.5.1 work for now?

**Comment by Gamerbolts** [comment]
Just found the mod and installed it, same issue. Has this been solved? Been nearly a year now.
